recipes/suchtree/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "SuchTree" %}
+{% set version = "1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/db/f0/75245602f31f80d18a7ffd4b4739e2eb442e46801c1c0ea430c0f002f2e7/suchtree-1.2.tar.gz"
```

Comment:
can you please use the other pypi URL and include the version jinja variable?

Reply:
I'm once again regretting including capital letters.